### ISSUE TITLE
[ BUG ] `New-FalconFirewallGroup` will not create a group following wiki example in PowerShell 5.1

### ISSUE THREAD

**Issue by bk-cs** [CrowdStrike/psfalcon#246]
**Describe the bug**
Errors are generated when you attempt to copy/paste the [example](https://github.com/CrowdStrike/psfalcon/wiki/Firewall-Management#creating-firewall-rule-groups) to create a new firewall group with a rule within PowerShell 5.1.

**To Reproduce**
Use PowerShell 5.1 and follow the wiki example.

**Expected behavior**

**Environment (please complete the following information):**
 - OS: Windows 10
 - PowerShell: 5.1
 - PSFalcon: 2.2.1

**Comment by bk-cs** [comment]
These errors are generated because of a difference in how PowerShell Desktop and PowerShell Core treat `[PSCustomObject]` and `[hashtable]` objects. In PowerShell Core, a `[hashtable]` can be used with `Select-Object` in the same way that `[PSCustomObject]` can, but in PowerShell Desktop, this doesn't work. 

To work around this problem until PSFalcon is updated, force `[hashtable]` objects to be `[PSCustomObjects]`. For instance, with the Firewall example, change `$Rules` from:
```powershell
$Rules = @(
    @{
        name = 'Block IP'
        description = 'Block outbound to example.com IP address'
        platform_ids = @( '0' )
        enabled = $true
        action = 'DENY'
        direction = 'OUT'
        address_family = 'IP4'
        protocol = '*'
        fields = @(
            @{
                name = 'network_location'
                type = 'set'
                values = @( 'ANY' )
            }
        )
        local_address = @(@{ address = '*'; netmask = 0 })
        remote_address = @(@{ address = '93.184.216.34'; netmask = 32 })
    }
)
```
To this instead:
```
$Rules = @(
    [PSCustomObject]@{
        name = 'Block IP'
        description = 'Block outbound to example.com IP address'
        platform_ids = @( '0' )
        enabled = $true
        action = 'DENY'
        direction = 'OUT'
        address_family = 'IP4'
        protocol = '*'
        fields = @(
            @{
                name = 'network_location'
                type = 'set'
                values = @( 'ANY' )
            }
        )
        local_address = @(@{ address = '*'; netmask = 0 })
        remote_address = @(@{ address = '93.184.216.34'; netmask = 32 })
    }
)
```

**Comment by bk-cs** [comment]
Resolved with v2.2.3 release
